.github/workflows/android-maestro.yml

```
@@ -121,6 +164,96 @@ jobs:
+                    } >> "$GITHUB_STEP_SUMMARY"
+
+            -   name: Save Gradle cache
+                if: steps.native-app-cache.outputs.cache-hit != 'true'
+                uses: actions/cache/save@v4
+                with:
+                    path: |
+                        ~/.gradle/caches
+                        ~/.gradle/wrapper
+                    key: gradle-${{ matrix.target }}-${{ runner.os }}-${{ hashFiles(format('{0}/android/gradle/wrapper/gradle-wrapper.properties', env.APP_DIR), format('{0}/android/**/*.gradle*', env.APP_DIR), format('{0}/package.json', env.APP_DIR), format('{0}/app.json', env.APP_DIR), 'yarn.lock') }}
+
+            -   name: Package native app
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)